packages/react-api/reference.md

```diff
@@ -25,6 +25,7 @@ Returns required default props for layers. It manages filtering and viewport cha
 | Param                               | Type                          | Description                                                               |
 | ----------------------------------- | ----------------------------- | ------------------------------------------------------------------------- | ---------------------------------------------------- |
 | props                               | <code>Object</code>           | Required default props for layers                                         |
+| props.uniqueIdProperty              | <code>string</code>           | Unique id property for the layer, default to `cartodb_id`                 |
```

Comment:
We must also add the uniqueIdProperty as output right?

Reply:
That block is actually the output

Reply:
sorry man, I didn't notice 👍🏻